client/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { act, render } from '@testing-library/react';
+import 'mobx-react-lite/batchingForReactDom';
```

Comment:
Looks like this line will be used in almost every test on components. Lets run it automatically:
https://jestjs.io/docs/en/configuration#setupfilesafterenv-array